File: Java/src/Iniciante/Solucao_1165.java
package Iniciante;

import java.util.ArrayList;
import java.util.Scanner;

public class Solucao_1165 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        boolean flag = true;

        var numeros = new ArrayList<Integer>();
        int n = 0;

        while (flag) {
            n = input.nextInt();
            if (n >= 1 && n <= 100) {
                flag = false;
            }
        }

        while (numeros.size() != n) {
            int x = input.nextInt();
            if (x > 1 && x <= Math.pow(10,7)) {
                numeros.add(x);
            }
        }
        for (Integer num : numeros) {
            boolean flagPrimo = true;
            for (int i = 2; i <= Math.sqrt(num); i++) {
                if (num % i == 0) {
                    flagPrimo = false;
                    break;
                }
            }
            if (flagPrimo) {
                System.out.println(num + " eh primo");
            } else {
                System.out.println(num + " nao eh primo");
            }

        }
    }
}

File: Python/Iniciante/Solucao_1165.py
import math


lista_numeros = []
flag = True
n = 0

while flag:
    n = int(input())
    if 1 < n <= 100:
        flag = False
while len(lista_numeros) != n:
    numeros = int(input())
    if 1 < numeros <= 10**7:
        lista_numeros.append(numeros)

for numeros in lista_numeros:
    flagPrimos = True
    for divisores in range(2, int(math.sqrt(numeros))+1):
        if numeros % divisores == 0:
            flagPrimos = False
            break
    if flagPrimos:
        print(f"{numeros} eh primo")
    else:
        print(f"{numeros} nao eh primo")


